src/style.css

```diff
@@ -0,0 +1,99 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirBold;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+body {
+  margin-top: 0;
+}
+
+.massiveBar {
+  margin: 20px 0;
+  position: relative;
+}
+
+.searchImg {
+  position: absolute;
+  width: 19px;
+  bottom: 26px;
+  left: 26px;
+}
+
+.bigSearchField {
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  color: #3d4e61;
+  background: white;
```

Comment:
if you want to change background color only, it is better to use the background-color CSS property
background is a shorthand property
https://developer.mozilla.org/en-US/docs/Web/CSS/background